src/main/java/com/spring/boot/amazon/helper/CsvFileParser.java

```diff
@@ -0,0 +1,29 @@
+package com.spring.boot.amazon.helper;
+
+import com.opencsv.CSVReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class CsvFileParser implements Parser {
+    @Override
+    public List parse(String csvFile, boolean isHeader) {
```

Comment:
в тебе уже є метод, який зчитує дані з файлу - `List read(String path)`
пропоную найперше - параметризувати тип даних що повертається, і потім використати результати його роботи для того щоб передати в метод:
```suggestion
    public List<Review> parse(List<String> data) {
```

Reply:
Я изначально собиралась считывать и парсить файл с помощью opencsv, но поскольку не смогла метод разбить на два отдельных, решила выполненить второй пункт ДЗ с помощью SimpleReader, а третий уже с использованием opencsv.
public List<Review> - так создание моделей только в следующей дз - "Add required entities and repositories" или я могу создать Review уже сейчас?